lib/elixir_tw/web/static/css/app.scss

```diff
@@ -0,0 +1,3 @@
+$icon-font-path: '/fonts/'; /* use fonts from priv/static/fonts/ */
+@import 'node_modules/font-awesome/scss/font-awesome';
+@import 'bootstrap'; /* imports node_modules/bootstrap-sass/assets/stylesheets/_bootstrap.scss */
```

Comment:
Use `//` comments everywhere

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#comment)